docs/static/index.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0;url=/v17">
```

Comment:
Can you think of a way to derive the `/v17` URL from is set in `gatsby-config.js`'s `options.currentVersion`?

Reply:
Yup! I just created a page at `src/index.js` that takes the `currentVersion` config option and adds the meta tag within a `Helmet` component. It seems to work just as well, but you still need to `build` and `serve` the site to see the effect locally.